databricks/koalas/ml.py

```diff
@@ -14,13 +14,24 @@
 # limitations under the License.
 #
 
-import pandas as pd
+from typing import List, Tuple, TYPE_CHECKING
+
 import numpy as np
+import pandas as pd
+import pyspark
+
 from pyspark.ml.feature import VectorAssembler
 from pyspark.ml.stat import Correlation
 
 
-def corr(kdf, method='pearson'):
+if TYPE_CHECKING:
+    import databricks.koalas as ks
+
+
+CORRELATION_OUTPUT_COLUMN = '_correlation_output'
+
+
+def corr(kdf: 'ks.DataFrame', method: str = 'pearson') -> pd.DataFrame:
```

Comment:
one thing that i think would really help with making this more understandable is to add an example (doctest?).